AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Requests/NotifyRequestorsCommandHandlerTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AllReady.Areas.Admin.Features.Requests;
+using AllReady.Models;
+using AllReady.Models.Notifications;
+using AllReady.Services;
+using MediatR;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Requests
+{
+    public class NotifyRequestorsCommandHandlerTests
+    {
+        [Fact]
+        public async Task ShouldSendSmsForEachRequestWithCorrectPhoneNumber()
```

Comment:
I like the approach here with an array of tests, however, it introduces logic into the test itself. I would recommend having a look at [theories and inline data](https://xunit.github.io/docs/getting-started-desktop.html#write-first-theory), which would likely lead you into something like [property and class data] for theories. 

The idea is: you write a test that accepts parameters, namely the test data and the expected result, then you simply provide the parameters through attributes, properties or instances of classes.

The benefit is that you end up with a test that is clear in what it is doing and your test data can easily grow without modifying the test.

If you are not testing for specific data points here, I would also suggest considering [GenFu](http://genfu.io/) for test/sample data generation as you would not have to new up objects on your own.